docs/sources/reference/commandline/cli.md

```diff
@@ -370,6 +370,41 @@ Currently supported options are:
 
         $ docker -d --storage-opt dm.blkdiscard=false
 
+ *  `dm.override_udev_sync_check`
+
+    Overrides the `udev` synchronization checks between devicemapper and udev.
+    `udev` is the device manager for the Linux kernel.
+
+    To view the `udev` sync support of a Docker daemon that is using the
+    devicemapper driver, run:
+
+        $ docker info
+	[...]
+	 Udev Sync Supported: true
+	[...]
+
+    When `udev` sync is supported, then devicemapper and udev can coordinate
+    the activation and deactivation of devices for containers.
+
+    When `udev` sync is `false`, a race conditions occurs between
```

Comment:
`udev` sync is not supported 

or

`udev` sync support is `false`

Ping @moxiegirl @jamtur01